src/RA_Dlg_RomChecksum.cpp

```diff
@@ -27,16 +27,29 @@ INT_PTR CALLBACK RA_Dlg_RomChecksum::RA_Dlg_RomChecksumProc(HWND hDlg, UINT nMsg
                 case IDC_RA_COPYCHECKSUMCLIPBOARD:
                 {
                     //	Allocate memory to be managed by the clipboard
-                    HGLOBAL hMem = GlobalAlloc(GMEM_MOVEABLE, g_sCurrentROMMD5.length() + 1);
-                    memcpy(GlobalLock(hMem), g_sCurrentROMMD5.c_str(), g_sCurrentROMMD5.length() + 1);
-                    GlobalUnlock(hMem);
+                    // wrapped just in case it goes out of scope early
+                    using GlobalOwner = std::unique_ptr<std::remove_pointer_t<HGLOBAL>, decltype(&::GlobalFree)>;
```

Comment:
This will cause the memory to get released when leaving the function, even if we hand it off to the clipboard. You probably need to change line 49 to `hMem.release()`.

Reply:
Ok